package.json

```diff
@@ -8,16 +8,18 @@
   },
   "devDependencies": {
     "casperjs": "~1.1.0-beta3",
+    "jscs": "~2.6.0",
+    "jshint": "~2.9.1",
```

Comment:
Weird.. well node_modules seems to be being removed automatically but it still fails: http://dpaste.com/0HVSSY6

I have npm version 3.4.1, so maybe that's why. It might be more strict now, and interpreting ~2.9.1 as version 2.9.1 or higher, i.e. 2.9.2, 2.9.3, which would exclude 2.9.1-rc1.